deploy/olm-catalog/community-kubevirt-hyperconverged/1.9.0/manifests/hco00.crd.yaml

```diff
@@ -2432,6 +2432,7 @@ spec:
                   Qps and burst values are taken from the annotation hco.kubevirt.io/tuningPolicy
                 enum:
                 - annotation
+                - HighBurst
```

Comment:
this is 1.9.0 CRD. It should not be supported here. Also, please avoid modify this file manually.